package.json

```diff
@@ -15,11 +15,11 @@
         "build-artifacts": "npm-run-all build make-dist",
         "unzip-artifacts": "unzip dist/artifacts/aws-cloudformation.zip -d dist/aws-cloudformation",
         "test": "mocha --timeout 3600000 -r ts-node/register test/*.spec.ts",
-        "lint-check-code": "node node_modules/ftnt-devops-ci/bin/ftnt-devops-ci.js c -fl  -F .eslintignore -L .eslintignore \"{*,**/*}{.js,.json,.ts}\"",
-        "lint-check-template": "node node_modules/ftnt-devops-ci/bin/ftnt-devops-ci.js c -f -F .eslintignore -L .eslintignore \"**/*.template.yaml\" --parser \"yaml\"",
+        "lint-check-code": "npx ftnt-devops-ci c -fl  -F .eslintignore -L .eslintignore \"{*,**/*}{.js,.json,.ts}\"",
```

Comment:
npx is redundant, should just be able to run `ftnt-devops-ci` since `node_modules/.bin` is added to $PATH during `npm run`
see https://docs.npmjs.com/cli/v7/using-npm/scripts#path

Reply:
done. uses of npx are removed.